libraries/SITL/examples/JSON/readme.md

```diff
@@ -72,6 +72,13 @@ Airspeed:
     airspeed (m/s)
 ```
 
+Motor RPM:
+
+```
+    motor:
+        rpm (rev/min)
+```
```

Comment:
Would be good to give a little more info here. Mention its a array per motor and give a example.

Reply:
Would this be sufficient?

Motor RPM:
```
    motor:
        rpm (rev/min) per motor in a single array. Limited to 12 motors. 
```
for example: ```"motor":{"rpm":[1000,1000,1000,...]}```

Reply:
perfect

Reply:
Done.